src/desktop/apps/artwork/client/errorModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import { ErrorModal } from "reaction/Components/Modal/ErrorModal"
+
+export const render = ({
+  headerText = "An error occurred",
```

Comment:
#minor but I believe this is the default so you technically don't have to specify